images/linux/scripts/installers/erlang.sh

```diff
@@ -8,20 +8,21 @@
 source $HELPER_SCRIPTS/install.sh
 
 source_list=/etc/apt/sources.list.d/eslerlang.list
+source_key=/usr/share/keyrings/eslerlang.gpg
 
 # Install Erlang
-echo "deb https://binaries.erlang-solutions.com/debian $(lsb_release -cs) contrib" > $source_list
-wget -q -O - https://binaries.erlang-solutions.com/debian/erlang_solutions.asc | apt-key add -
+wget -q -O - https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc | gpg --dearmor > $source_key
+echo "deb [signed-by=$source_key]  https://packages.erlang-solutions.com/ubuntu $(lsb_release -cs) contrib" > $source_list
```

Comment:
nitpick: use the `getOSVersionLabel` helper from the `os.sh` helper to get a codename